crates/bevy_state/src/state/states.rs

```diff
@@ -64,4 +64,7 @@ pub trait States: 'static + Send + Sync + Clone + PartialEq + Eq + Hash + Debug
     /// Used to help order transitions and de-duplicate [`ComputedStates`](crate::state::ComputedStates), as well as prevent cyclical
     /// `ComputedState` dependencies.
     const DEPENDENCY_DEPTH: usize = 1;
+
+    /// Should scoped entities be enabled for this state?
```

Comment:
This should probably backlink to `StateScoped` or `App::enable_state_scoped_entities`.